src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+}
+
+.form_box {
+  margin-top: 20px;
+}
+
+.search {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus {
+  font-weight: bold;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
try to use `background-color` property instead of `background` – it causes either icon to disappear if user focuses on it